sktime/classification/feature_based/_random_interval_classifier.py

```diff
@@ -49,6 +49,18 @@ class RandomIntervalClassifier(BaseClassifier):
     See Also
     --------
     RandomIntervals
+
```

Comment:
Dame as above, I think examples need to be in their own section rather than in See Also.